files/default/README.md

```diff
@@ -0,0 +1,14 @@
+# Unit Testing
+
+Make a python virtual environment, install the requirements with:
+
+    mkvirtualenv hatool -p python2
```

Comment:
no every one uses virtualenvwrapper, so I would suggest to use `virtualenv` command